src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -260,7 +260,7 @@ def default_api_version(self):
         ResourceType.MGMT_CONTAINERREGISTRY: '2019-05-01',
         ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2020-11-01', {
             'container_services': '2017-07-01',
-            'open_shift_managed_clusters': '2019-10-27-preview'
+            'open_shift_managed_clusters': '2019-09-30-preview'
```

Comment:
If we do not remove the [definition](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/_client_factory.py#L68) in file "_client_factory.py", any definition in file "_shared.py" would be overridden. Besides, the usage of api version for openshift command group has not been changed since this (PR#13618)(https://github.com/Azure/azure-cli/pull/13618).